packages/server/graphql/mutations/updateJiraDimensionField.ts

```diff
@@ -77,12 +77,25 @@ const updateJiraDimensionField = {
     if (!auth) {
       return {error: {message: 'Not authenticated with Jira'}}
     }
-    const {accessToken} = auth
-    const manager = new AtlassianServerManager(accessToken)
-    const fields = await manager.getFields(cloudId)
-    const field = fields.find((field) => field.name === fieldName)
-    if (!field) return {error: {message: 'Invalid field name'}}
-    const {id: fieldId, schema} = field
+
+    let selectedField
```

Comment:
great fix! the only nit is that `selectedField` has an implicit type of `any`, which is pretty common among let statements.
we could cast it to a `Partial<JiraField>`, but partials mean you always have to check nullability.

instead, i'd propose just making a little helper function that keeps the return values strict. that way if anything changes (e.g. the JiraField interface) then we'll get errors for free!

example refactor here: https://github.com/ParabolInc/parabol/pull/5069/files

Reply:
right, that's a good point! 🙏